dip-0024.md

```diff
@@ -551,7 +551,7 @@ The internal Dash message name is `qrinfo` and the format of the message is:
 | quorumSnapshotAtHMinus4C | CQuorumSnapshot | Variable | Returned only if extraShare is on. See below for sub-message contents.
 | mnListDiffAtHMinus4C | CSimplifiedMNListDiff | Variable | Returned only if extraShare is on. As in [DIP-4](https://github.com/dashpay/dips/blob/master/dip-0004.md).
 | num(lastQuorumHashPerIndex) | compactSize uint | 1-9 | Number of elements in lastCommitmentPerIndex
-| lastCommitmentPerIndex | uint256_t[] | 32 * num(lastCommitmentPerIndex)  | Contains the most recent commitment for each quorumIndex. Ordered by quorumIndex.
+| lastCommitmentPerIndex | CFinalCommitment[] | Variable  | Contains the most recent commitment for each quorumIndex. Ordered by quorumIndex.
```

Comment:
Is `CFinalCommitment` the same as the `qfcommit` from https://github.com/dashpay/dips/blob/master/dip-0006.md#6-finalization-phase?

Reply:
Yes, switching to `qfcommit` instead.